proca/lib/proca/target.ex

```diff
@@ -0,0 +1,50 @@
+defmodule Proca.Target do
+  @moduledoc """
+  Target represents a target of a campaign, and contains information about the Target
+  """
+
+  use Ecto.Schema
+  alias Proca.{Repo, Target}
+  import Ecto.Changeset
+  import Ecto.Query
+
+  @primary_key {:id, Ecto.UUID, autogenerate: true}
+
+  schema "targets" do
+    field :name, :string
+    field :external_id, :string
+    field :area, :string
```

Comment:
what's the default? (double checking it can be null)

Reply:
name, external_id and emails are required. Area and Fields can be null